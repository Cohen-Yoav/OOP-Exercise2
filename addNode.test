# addNode method Testing file

# check that the node was added to the graph
CreateNode n1 5
CreateGraph graph1
AddNode graph1 n1
ListNodes graph1

# check that adding a node 2 times only add it once to the graph
CreateNode n2 2
AddNode graph1 n2
AddNode graph1 n2
ListNodes graph1

