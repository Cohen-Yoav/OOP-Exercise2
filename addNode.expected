# addNode method Testing file

# check that the node was added to the graph
created node n1 with cost 5
created graph graph1
added node n1 to graph1
graph1 contains: n1

# check that adding a node 2 times only add it once to the graph
created node n2 with cost 2
added node n2 to graph1
graph1 contains: n1 n2

